docs/stencil-docs/storefront-customization/theme-accessibility.md

```diff
@@ -0,0 +1,225 @@
+# Developing Themes for Accessibility
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Image alt text](#image-alt-text)
+- [Text accessibility](#text-accessibility)
+- [Keyboard accessibility](#keyboard-accessibility)
+- [Customization](#customization)
+- [Related resources](#related-resources)
+</div>
+
+Designing for accessibility means being inclusive to the needs of all users. This article outlines the basic accessibility requirements BigCommerce would recommend for all themes.
+
+We have included four accessibility best practices and examples that will help you develop a theme that meets the needs of the broadest range of abilities.
+
+### Prerequisites
+* [A BigCommerce store](https://support.bigcommerce.com/s/article/Starting-a-Bigcommerce-Trial).
+* A store theme. (The following examples use the Cornerstone theme.)
+
+## Image alt text
+Alternative (alt) text is an image description read aloud by a screen reader. Adding alt text increases the content's accessibility to visually-impaired users. All HTML `<img>` elements should use the `alt` attribute to provide a text description of the image.
+
+**Store logo example**
+
+In Cornerstone, go to `templates/components/common/store-logo.html`. You will see the following code:
+
+```html
+ <a href="{{urls.home}}" class="header-logo__link">
+    {{#if settings.store_logo.image}}
+        {{#if theme_settings.logo_size '===' 'original'}}
+            <img class="header-logo-image-unknown-size" src="{{getImage settings.store_logo.image 'logo_size'}}" alt="{{settings.store_logo.title}}" title="{{settings.store_logo.title}}">
+        {{else}}
+            <div class="header-logo-image-container">
+                <img class="header-logo-image" src="{{getImage settings.store_logo.image 'logo_size'}}" alt="{{settings.store_logo.title}}" title="{{settings.store_logo.title}}">
+            </div>
+        {{/if}}
+    {{else}}
+        <span class="header-logo-text">{{settings.store_logo.title}}</span>
+    {{/if}}
+</a>
+```
+Add or update the image alt text to provide descriptive text for your store logo.
+
+```html
+alt = "BigCommerce store logo" 
+```
+
+NOTE: You can update the `<title>` tag with the same text as the `<alt>` tag to create a tooltip, or remove it if you do not need it.
```

Comment:
This note should be structured like the note in the "Color Contrast" section and wrapped in the appropriate `<div>` callouts.